Assets/SEE/Utils/VersionControlSystems.cs

```diff
@@ -0,0 +1,144 @@
+using UnityEngine;
+using LibGit2Sharp;
+using System.Linq;
+using System.IO;
+
+/// <summary>
+/// Contains all version control systems as subclasses, which derives from IVersionControl,
+/// so that all have the same functionality.
+/// </summary>
+public class VersionControlSystems : MonoBehaviour
```

Comment:
See my above note on SwitchVersionControlSystems: This must be nested in a namespace and it should rather be somewhere else in a new version-control layer, not part of SEE.Utils.

Reply:
It's fixed.